src/common/options.cc

```diff
@@ -3949,7 +3949,7 @@ std::vector<Option> get_global_options() {
     .set_description("When tcmalloc and cache autotuning is enabled, set the minimum amount of memory used for caches."),
 
     Option("osd_memory_cache_resize_interval", Option::TYPE_FLOAT, Option::LEVEL_DEV)
-    .set_default(1)
+    .set_default(5)
```

Comment:
So I did some testing at one point on this and ended up favoring a lower cache resize interval to keep ahead of large changes in cache growth (like when onodes are first populated in the cache).  Given our current double-caching of onodes in the onode cache and block cache it's useful to avoid reads for as long as possible.

Having said that, this would lower CPU consumption a little bit in the mempool thread.  What was your reasoning for increasing it?